packages/langium/test/dependency-injection.test.ts

```diff
@@ -352,18 +352,9 @@ describe('The inject result', () => {
         expect('b' in obj).toBe(false);
     });
 
-    test('should be extensible', () => {
+    test('should not be extensible', () => {
         const obj: any = inject({});
-        expect(Object.isExtensible(obj)).toBe(true);
-        expect(obj.a).toBeUndefined();
-        expect(() => obj.a = 1).not.toThrow();
-        expect(obj.a).toBe(1);
-    });
-
-    test('should be sealable', () => {
-        const obj: any = Object.seal(inject({}));
-        expect(Object.isExtensible(obj)).toBe(false);
-        expect(() => (obj.a = 1)).toThrowError('Cannot define property a, object is not extensible');
+        expect(() => (obj.a = 1)).toThrowError('Cannot set property on injected service container');
```

Comment:
Looks a bit weird because `a` was not initially set to a value.
I would assume that any property that is not defined on this object is inaccessiable.

Reply:
Without the `set` override on the proxy it was previously completely valid to set the value. This test ensures that it's not possible.